lib/Entity/Display.php

```diff
@@ -820,8 +853,18 @@ private function setConfig()
 
                 $displayProfile = $this->displayProfileFactory->getUnknownProfile($this->clientType);
             }
+            if (isset($this->overrideConfig)) {
```

Comment:
I see what you're trying to do here, but I think the logic needs to be after you've loaded the display profile settings - something like:

```
// load my config

// load in the display profile config

// if I have been asked to in options, merge my display overrides with the config i have from the display profile